index.d.ts

```diff
@@ -75,6 +75,26 @@ declare const isStream: {
 	```
 	*/
 	transform(input: unknown): input is stream.Transform;
+
+	/**
+	@returns The [`type of stream`](https://nodejs.org/api/stream.html#stream_types_of_streams).
+
+	@example
+	```
+	import * as fs from 'fs';
+	import isStream = require('is-stream');
+
+	isStream.type(fs.createReadStream('unicorn.png'));
+	//=> readable
+
+	isStream.type(fs.createWriteStream('unicorn.png'));
+	//=> writable
+
+	isStream.type({});
+	//=> no-stream
```

Comment:
I think returning `undefined` in this case would be better.

Reply:
done 64d915b